plugin/webdevicons.vim

```diff
@@ -117,6 +117,11 @@ function s:getDistro()
     endif
     return s:distro
   endif
+ " check for freeBSD
+  if system('uname -s') ==# "FreeBSD\n"
```

Comment:
Why do you need to call the system function again?
This may cause Vim to lose performance.

Reply:
At first glance, it was the simplier way achieving it. i'll give a try to _has feature bsd_ rather than uname